rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -93,22 +107,33 @@ private void givenExistingRepository(String repoUrl) {
         entity = repositoryEntityUtils.create(repoUrl);
     }
 
+    private void givenRandomRepositories(String repoUrlBase, int amount) {
+        IntStream.range(0, amount).forEach(i -> {
+            var repoUrl = repoUrlBase.replaceAll(GIT_EXTENSION, rand.nextInt() + GIT_EXTENSION);
```

Comment:
Why do we need the repository url be random? Having github.com/user/repo1.git, github.com/user/repo2.git and github.com/user/repo3.git will suffice and as we're deleting all the repositories, there won't be conflicts.